edb/server/dbview/dbview.pyx

```diff
@@ -174,6 +174,9 @@ cdef class Database:
                 metrics.background_errors.inc(
```

Comment:
^^ Also we should probably stop using `debug.dump` directly in code and do proper `logging.error` or whatnot. If it's not printing when debugging that should be debugged